### PR TITLE
Lesson json support

### PR DESCRIPTION
**Pull request by Jollybomber**

### CODE REVIEW

File: src/main/java/seedu/address/model/person/Person.java [hunk 1]
```diff
@@ -88,7 +94,8 @@ public boolean isSamePerson(Person otherPerson) {
         }
 
         return otherPerson != null
-                && otherPerson.getName().equals(getName());
+                && otherPerson.getName().equals(getName())
+                && otherPerson.getId() == getId();
```
Comment (estellelim) [inline src/main/java/seedu/address/model/person/Person.java:98]: this method checks for duplicate persons in the address book when adding tutor or tutees, if you check for id which are different for every newly created person object, it won't return true

**Review by estellelim — CHANGES_REQUESTED** [review]

File: src/main/java/seedu/address/model/person/Person.java (hunk 1 above)
Comment (Jollybomber) [reply src/main/java/seedu/address/model/person/Person.java]: Removed the getID() line

File: src/main/java/seedu/address/logic/commands/DeleteLessonCommand.java [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+            return true;
+        }
+
+        // instanceof handles nulls
+        if (!(other instanceof DeleteLessonCommand)) {
+            return false;
+        }
+
+        DeleteLessonCommand otherDeleteCommand = (DeleteLessonCommand) other;
+        return tutorIndex.equals(otherDeleteCommand.tuteeIndex);
```
Comment (estellelim) [inline src/main/java/seedu/address/logic/commands/DeleteLessonCommand.java:78]: Is this the intended behaviour?

**Review by estellelim — APPROVED** [review]

File: src/main/java/seedu/address/logic/commands/DeleteLessonCommand.java (hunk 2 above)
Comment (Jollybomber) [reply src/main/java/seedu/address/logic/commands/DeleteLessonCommand.java]: Nope, fixed it

**Pull request merged**